lib/public/components/qcFlags/isRunSubjectToQc.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license
+ * Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+ * See http://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+ * All rights not expressly granted are reserved.
+ *
+ * This software is distributed under the terms of the GNU General Public
+ * License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { RunQualities } from '../../domain/enums/RunQualities.js';
+
+/**
+ * Array of names of tags which prevent run to be subject to QC
+ */
+const RUN_TAGS_DISABLING_QC = ['Not for physics'];
+
+/**
+ * States whther run is subject to QC
+ *
+ * @param {Run} run a run
+ * @return {boolean} true if run is NOT subject to QC, false otherwise
+ */
+export const isRunNotSubjectToQc = ({ runQuality, tags }) => runQuality === RunQualities.BAD
+    || tags.find(({ text }) => RUN_TAGS_DISABLING_QC.includes(text));
+
+/**
+ * Get reason why a run is not subject to QC
+ *
+ * @param {Run} run a run
+ * @return {string} reason
+ */
+export const getRunNotSubjectToQcReason = ({ runQuality, tags }) => {
+    let reason = '';
+    if (runQuality === RunQualities.BAD) {
+        reason += `Quality of the run was changed to ${RunQualities.BAD} so it is no more subject to QC`;
+    }
+    if (tags.find(({ text }) => RUN_TAGS_DISABLING_QC.has(text))) {
+        reason += 'Run was tags as not for physics so it is no more subject to QC';
+    }
+    return reason;
```

Comment:
I would rather return `reasons`, an array and then joint them as needed in caller. 
Because you might want to display them as list, or concatenate depending on the context.
Here, when run has both it displays `...to QCRun was tags...`

Reply:
fixed